Beginner tutorial code fixes, upleveling, and Remix deployments (#534)
smartcontractkit/documentation: docs/Introduction/tutorials/beginners-tutorial.md

In `HelloWorld`, the constructor takes in a `string` as a parameter and sets the `message` state variable to that string.

```solidity

> {% include snippets/Tutorials/beginner-constructor.sol %}

Reviewer: Minimizing this example is fine, but I don't think it needs to be in the snippets folder.

Interfaces define functions without their implementation, leaving inheriting contracts to define the actual implementation themselves. This makes it easier to know what functions to call in a contract. Here's an example of an interface:

```solidity
{% include samples/Tutorials/Test.sol %}
```
<div class="remix-callout">
  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/Tutorials/Test.sol" target="_blank" class="cl-button--ghost">Deploy this contract using Remix ↗</a>
</div>

> For this example, `override` is necessary to use in the `Test` contract function since it is overriding the base function contained in the `numberComparison` interface. `pure` is also used in this example: since the `isSameNum` function in the `Test` contract does not return a storage variable, the contract uses `pure` instead of `view`.  

Reviewer: ```suggestion
For this example, `override` is necessary in the `Test` contract function because it overrides the base function contained in the `numberComparison` interface. The contract uses `pure` instead of `view` because the `isSameNum` function in the `Test` contract does not return a storage variable. 
```
